src/platform/silabs/SiWx917/wifi/ethernetif.cpp

```diff
@@ -192,10 +191,13 @@ static SemaphoreHandle_t ethout_sem;
  ******************************************************************************/
 static err_t low_level_output(struct netif * netif, struct pbuf * p)
 {
+    sl_wifi_buffer_t * buffer;
+    sl_si91x_packet_t * packet;
+    sl_status_t status = SL_STATUS_OK;
     void * rsipkt;
     struct pbuf * q;
     uint16_t framelength;
-
+    SILABS_LOG("LWIP : low_level_output");
```

Comment:
This log must be removed or kept under `WIFI_DEBUG_ENABLED` flag

Reply:
Added the log under WIFI_DEBUG_ENABLED flag.